src/components/FilterBar.jsx

```diff
@@ -21,13 +29,31 @@ const amounts = {
   5000: '> 5000',
 };
 
+const sortingOptions = {
+  deadlineSoon: 'Deadline (Earliest to Latest)',
+  deadlineLatest: 'Deadline (Latest to Earliest)',
+  amountLow: 'Amount (Low to High)',
+  amountHigh: 'Amount (High to Low)',
+};
+
 function FilterBar() {
+  const classes = useStyles();
   return (
-    <div>
-      <FilterDropDown label="Major" items={majors} />
-      <FilterDropDown label="Grade" items={grades} />
-      <FilterDropDown label="Amount" items={amounts} />
-    </div>
+    <Grid container spacing={2} className={classes.root}>
+      <Grid item>
+        <FilterDropDown label="Major" items={majors} defaultSelect="" />
+        <FilterDropDown label="Grade" items={grades} defaultSelect="" />
+        <FilterDropDown label="Amount" items={amounts} defaultSelect="" />
+      </Grid>
+      <Grid item>
+        <FilterDropDown
+          label="Sorting"
```

Comment:
Maybe **Sort by** would be better. 

To stick to the Figma design though, this should probably be separated. E.g. 
```jsx
<Grid item>
  Sort by
  <FilterDropdown...>
</Grid>
```

Then to get it to center vertically you style its parent container (`<Grid item>` in this case) to have the `display: 'flex'` and `alignItems: 'center'` properties